Dockerfile

```diff
@@ -43,7 +43,6 @@ ENV PKG_CONFIG_PATH="/usr/local/lib/pkgconfig:$PKG_CONFIG_PATH"
 # ==================================[ BUILD ]========================================
 WORKDIR /SOR
 
-RUN cabal update
```

Comment:
We could remove this, but this is not related to the docker-compose topic, so I am going to remove this change from the PR.